app/scripts/controllers/push-platform-notifications/push-platform-notifications.ts

```diff
@@ -0,0 +1,229 @@
+import {
```

Comment:
Why is this controller implemented directly in the extension as opposed to `/core`?

Reply:
Yes, we've already discussed this issue for all the other controllers we're using to import the new feature dedicated to notifications. You're absolutely right, and ultimately, we want to implement this and the other controllers in core, but currently, for reasons of speed in implementation, we've decided to work directly in the extension